frontend/integration-tests/tests/environment.scenario.ts

```diff
@@ -136,17 +148,37 @@ describe('Interacting with the environment variable editor', () => {
       const envPrefix = 'testcm';
       await environmentEditor(Actions.addFrom, resourceName, envPrefix);
       await environmentView.isLoaded();
-      await validateValueFrom(resourceName, envPrefix);
+      await validateValueFrom(resourceName, envPrefix, true);
```

Comment:
nit: from a readability standpoint, trailing `bool` values tends to be confusing. 
Hate to ask to rework to an object, potentially could:

```js
const isPresent = true
await validateValueFrom(name, prefix, isPresent);
await validateValueFrom(name, prefix, !isPresent);
```